src/style.css

```diff
@@ -0,0 +1,123 @@
+html {
+  --font-color-main: #060b35;
+  --font-color-second: #616070;
+  --brand-main-color: #00acdc;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.product-card {
+  display: flex;
```

Comment:
add padding to the whole product card, so you won't need to add horizontal margins to every element inside of it